pyMBE.py

```diff
@@ -1673,7 +1766,29 @@ def define_residue(self, name, central_bead, side_chains):
         self.df.at [index,'pmb_type'] = 'residue'
         self.df.at [index,'central_bead'] = central_bead
         self.df.at [index,('side_chains','')] = side_chains
-        return 
+        return
+
+    def delete_molecule_entry(self, molecule_name):
+        """
+        Deletes a molecule entry from the DataFrame if it exists.
+
+        Args:
+            molecule_name (`str`): The name of the molecule to delete.
+        """
+        if molecule_name in self.df["name"].values:
+            self.df = self.df[self.df["name"] != molecule_name].reset_index(drop=True)
+
+    def delete_particle_entry(self, particle_name): 
+
+        """ 
+        Deletes a particle entry from the DataFrame if it exists. 
+        
+        Args: 
+            particle_name (`str`): The name of the particle to delete. 
+
+        """ 
+        if particle_name in self.df["name"].values: 
+            self.df = self.df[self.df["name"] != particle_name].reset_index(drop=True) 
```

Comment:
@1234somesh I think that `delete_molecule_entry` and  `delete_particle_entry`  should be deleted from this PR. We have already replaced them by `delete_entries_in_df`  in your other PR  #112 and they are no longer needed here.